astroquery/mast/tests/test_mast.py

```diff
@@ -177,6 +180,58 @@ def zcut_download_mockreturn(url, file_path):
     return
 
 
+###########################
+# MissionSearchClass Test #
+###########################
+
+
+def test_missions_query_region_async(patch_post):
+    responses = mast.MastMissions.query_region_async(regionCoords, radius=0.002, sci_pi_last_name='GORDON')
+    assert isinstance(responses, MockResponse)
+
+
+def test_missions_query_object_async(patch_post):
+    responses = mast.MastMissions.query_object_async("M101", radius="0.002 deg")
+    assert isinstance(responses, MockResponse)
+
+
+def test_missions_query_object(patch_post):
+    result = mast.MastMissions.query_object("M101", radius=".002 deg")
+    assert isinstance(result, Table)
```

Comment:
Could also add an assertion that there is content within the table with `assert len(Table) > 0`